tests/tensorflow2/test_config_validator_tensorflow.py

```diff
@@ -0,0 +1,15 @@
+# Future
+from __future__ import print_function
+
+# Third Party
+from packaging import version
+
+
+def test_supported_tensorflow_version(monkeypatch):
+    import smdebug.tensorflow.singleton_utils
+    import smdebug.tensorflow.utils
+
+    with monkeypatch.context() as m:
```

Comment:
Don't know much about `monkeypatch.context`. But the way I set temporary env variables with monkeypatch is using `monkeypatch.setenv`.

Example: https://github.com/awslabs/sagemaker-debugger/blob/master/tests/profiler/core/test_profiler_config_parser.py#L31